changelogs/fragments/4108-nmcli-support-modifcation-without-type-param.yml

```
@@ -0,0 +1,2 @@
+bugfixes:
```

To me this rather looks like a feature:
```suggestion
minor_changes:
```

---

done